app/controllers/concerns/ownership_concern.rb

```diff
@@ -0,0 +1,27 @@
+
+require 'forwardable'
```

Comment:
The ruby `forwardable` module allows you to delegate certain methods to other objects in your application.
Say you want to create a Queue system which uses an Array to queue items, one way to loop through would be to 
```ruby
def each(*item, &block)
queue.each{ |x|
 yield x if block_given?
}
end
```
However with the forwardable module, you could just delegate that method call to the appropriate object and let it handle it that way. So for example you could `def_delegator @queue, :each, :first, :last, :max, :min` and other array methods you would like to delegate to.
In my case `def_delegator self, :resource_name` which allowed me to call the class_method `resource_name` inside an instance_method